kas-iot2050-boot-m2.yml

```diff
@@ -0,0 +1 @@
+kas-iot2050-boot-pg2.yml
```

Comment:
Should we add new entry for m.2 in `kas-container menu`?

Reply:
has already add the entry for m2